new/LIT6745Dersan.xml

```diff
@@ -0,0 +1,63 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6745Dersan" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ድርሳን፡ በእንተ፡ ዘይቤ፡ ውስተ፡ ወንጌል፡ ንዑ፡ ኀቤየ፡ ሥሩሓን፡ ወክቡዳን፡ ወአነ፡ አዐረፈክሙ፡</title>
+                <title xml:lang="en" corresp="#t2">Homily on Mt. 11, 28 for the sixth Saturday in Lent</title>
```

Comment:
since there is no xml:id="t2", corresp="#t2" will lead nowhere - I would suggest giving the English as xml:id="t2", because it is not a translation of the Ethiopic